install/install_openadapt.ps1

```diff
@@ -271,4 +271,4 @@ RunAndCheck "alembic upgrade head" "alembic upgrade head"
 RunAndCheck "pytest" "run OpenAdapt tests"
 Write-Host "OpenAdapt installed successfully"
 
-################################   SCRIPT    ################################
+################################   SCRIPT    ################################ 
```

Comment:
@dianzrong It will be great to remove an extra white space here at Line 274 and push a new commit so that there is no change in ps1

Reply:
Resolved in: [42b5e42](https://github.com/MLDSAI/OpenAdapt/pull/244/commits/42b5e42fd8343878a955a9fb81990301f4851843)